services-custom/s3-transfer-manager/src/it/java/software/amazon/awssdk/transfer/s3/S3TransferManagerUploadPauseResumeIntegrationTest.java

```diff
@@ -139,13 +163,16 @@ void pause_fileChanged_resumeShouldStartFromBeginning() throws Exception {
         assertThat(resumableFileUpload.totalParts()).isNotEmpty();
         verifyMultipartUploadIdExists(resumableFileUpload);
 
+        byte[] originalBytes = Files.readAllBytes(largeFile.toPath());
         byte[] bytes = "helloworld".getBytes(StandardCharsets.UTF_8);
         Files.write(largeFile.toPath(), bytes);
 
-        FileUpload resumedUpload = tmCrt.resumeUploadFile(resumableFileUpload);
+        FileUpload resumedUpload = transferManager.resumeUploadFile(resumableFileUpload);
         resumedUpload.completionFuture().join();
         verifyMultipartUploadIdNotExist(resumableFileUpload);
         assertThat(resumedUpload.progress().snapshot().totalBytes()).hasValue(bytes.length);
+
+        Files.write(largeFile.toPath(), originalBytes);
```

Comment:
Unintentional change?

Reply:
we only overwrite it in this test, so dont need to write back for other tests